.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -34,6 +34,8 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
 
             // Push, save the MOD string
             bool hasBefore = false, hasAfter = false, hasSince = false, hasYearAfter = false;
+            // "Include" means MOD string like "on or later than", "earlier than or in"
+            bool hasInclude = false;
```

Comment:
A better name would be hasInclusiveModifier.

Reply:
The code comment should also describe the intent of if. Not just a couple specific examples.

Reply:
Nit: Add blank lines before code comments.

Reply:
Fixed